close #10 add tailwind css and configure files.

OzerOzturk opened the pull request.

OzerOzturk committed:
close #10 add tailwind css and configure files.

maqalaqil reviewed (changes requested):
this is a very big PR @OzerOzturk please write a description for it and for everything you have added to help us Review it.

Iseluin commented on package.json:
```diff
@@ -47,7 +49,10 @@
     "eslint-plugin-react": "^7.29.4",
     "eslint-plugin-react-hooks": "^4.3.0",
     "husky": "^7.0.0",
-    "prettier": "^2.6.0"
+    "postcss": "^8.4.12",
+    "postcss-cli": "^9.1.0",
+    "prettier": "^2.6.0",
```
Why was prettier deleted and added again here? Please explain. It might have happened automatically when you added the tailwind library.

OzerOzturk replied:
When I added tailwind css, this happened automatically.

OzerOzturk commented:
I can explain step by step what I did in this pr.

1. eslintrc.json --> I made changes because of the eslint error/warning  I face while rendering. Such as "import/no-default-export". 
2. package.json --> Required packages have been installed to use tailwind css.
3. postcss.config.js --> this file was created while the configuration of tailwind css.
4. App.css --> We don't use app.css  so I deleted default styles.
5. App.jsx --> I deleted default codes and added simple div and h1 to show "hello world" using tailwind css. 
6. store.js --> I made the changes because of the eslint error/warning.
7. index.css --> we have to make some configurations to use tailwind css, this is why I edited it.
8. index.js --> this change is related step 6 (eslint error/warning)
9. tailwind.config.js --> this file was created while the configuration of tailwind css.

Iseluin reviewed (changes requested):
Eslint helps us to write clean and production-level code, so unless we completely have to, let us not touch eslintrc.json file (don't add eslint exceptions). 

If you want, you can delete the whole store and slices since we will configure our own redux store.

All in all, this PR looks good and will be merged after you delete the eslint exception and adjust your code to it.

OzerOzturk committed:
close #10 retrieve eslint exceptions.
